### PR TITLE
NGWIN.PicPick version 6.0.0

### PR DESCRIPTION
**Pull request by DanTheMan827**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/51099)

### CODE REVIEW

File: manifests/p/picpick/picpick/6.0.0/picpick.picpick.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: picpick.picpick
```
Comment (Trenly) [inline manifests/p/picpick/picpick/6.0.0/picpick.picpick.yaml:4]:
```suggestion
PackageIdentifier: NGWIN.picpick
```

File: manifests/p/picpick/picpick/6.0.0/picpick.picpick.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: picpick.picpick
```
Comment (Trenly) [inline manifests/p/picpick/picpick/6.0.0/picpick.picpick.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: NGWIN.picpick
```

File: manifests/p/picpick/picpick/6.0.0/picpick.picpick.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: picpick.picpick
```
Comment (Trenly) [inline manifests/p/picpick/picpick/6.0.0/picpick.picpick.installer.yaml:4]: I think we should move this package to align with `Publisher.Package`
```suggestion
PackageIdentifier: NGWIN.picpick
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/51099#pullrequestreview-906315966

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You also need to update the publisher folder from `picpick` to `NGWIN` by using Visual Studio Online (press the `.` (dot) on your keyboard on this PR) or by using GitHub Desktop.

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Make sure to update the file names as well (to match PackageIdentifier)

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You need to fix the capitalization in the file name.

from `pickpick` to `PicPick`.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**